homeworks/01-twitch/README.md

```diff
@@ -60,6 +62,7 @@ public interface StreamingPlatform {
      *
      * @param username the username of the watcher
      * @param content  the content to watch
+     * @throws IllegalArgumentException if any of the parameters are null or empty
```

Comment:
Maybe change it to "if any of the parameters are null (or a String is empty)" to avoid ambiguity, what does empty mean for a Content.